containers/aesmd/Dockerfile

```diff
@@ -0,0 +1,43 @@
+# This Dockerfile starts the AESM service, which is used by Gramine hardware tests.
+FROM ubuntu:20.04
+
+ARG commit_id
+
+LABEL commit_id=${commit_id}
+
+# Install dependencies
+RUN apt-get update
+RUN DEBIAN_FRONTEND=noninteractive  apt-get install -y \
+    wget \
+    gnupg2
+
+RUN wget -qO - https://download.01.org/intel-sgx/sgx_repo/ubuntu/intel-sgx-deb.key | apt-key add -
+RUN echo 'deb [arch=amd64] https://download.01.org/intel-sgx/sgx_repo/ubuntu focal main' > /etc/apt/sources.list.d/intel-sgx.list
+
+RUN apt-get update
+
+# TODO: Update Conclave to Intel SGX SDK 2.18 and remove package versions: https://r3-cev.atlassian.net/browse/CON-1243
+RUN DEBIAN_FRONTEND=noninteractive apt-get install -y \
+    sgx-aesm-service=2.17.100.3-focal1 \
```

Comment:
In theory we should only need this package, right? Can you try removing the others and see if it still works.

Reply:
I think I got issues with dependencies. I.e. if a package has dependencies, you need to specify the version for the dependencies too.
Same for all the libsgx* ones. We only need one or two, in theory.
 I will double check again.

Reply:
OK, in that case, can you comment on which packages are actually dependencies on the aesm service, and thus can be removed once we move onto the latest version.

Reply:
I confirm that dependencies' versions are not following the "established" dependent version. 
I.e. if I run this
```
sudo apt-get install libsgx-quote-ex=2.17.100.3-focal1 libsgx-dcap-ql=1.14.100.3-focal1
```
here is what I get
```
...
Setting up libsgx-ae-qe3 (1.15.100.3-focal1) ...
Setting up libsgx-dcap-quote-verify (1.15.100.3-focal1) ...
Setting up libsgx-ae-le (2.18.100.3-focal1) ...
Setting up libsgx-quote-ex (2.17.100.3-focal1) ...
Setting up libsgx-ae-id-enclave (1.15.100.3-focal1) ...
Setting up libsgx-launch (2.18.100.3-focal1) ...
Setting up libsgx-ae-qve (1.15.100.3-focal1) ...
Setting up libsgx-enclave-common (2.18.100.3-focal1) ...
Setting up libsgx-ae-epid (2.18.100.3-focal1) ...
Setting up libsgx-ae-pce (2.18.100.3-focal1) ...
Setting up libsgx-aesm-launch-plugin (2.18.100.3-focal1) ...
Setting up libsgx-urts (2.18.100.3-focal1) ...
Setting up libsgx-pce-logic (1.15.100.3-focal1) ...
Setting up libsgx-qe3-logic (1.15.100.3-focal1) ...
Setting up libsgx-dcap-ql (1.14.100.3-focal1) ...
Setting up libsgx-aesm-pce-plugin (2.18.100.3-focal1) ...
Setting up libsgx-aesm-epid-plugin (2.18.100.3-focal1) ...
Setting up libsgx-aesm-ecdsa-plugin (2.18.100.3-focal1) ...
Setting up libsgx-aesm-quote-ex-plugin (2.18.100.3-focal1) ...

```
From these I just removed the epid ones.
I was already bitten by some libraries being on 2.18 and AESM not working, so I think it is correct to set all those dependencies as I have done.

I suggest to put a TODO asking to replace all of these with what was there before, i.e. the default instructions (as suggested by Intel SGX SDK, "Intel® SGX Application User")
https://download.01.org/intel-sgx/latest/dcap-latest/linux/docs/Intel_SGX_SW_Installation_Guide_for_Linux.pdf

```
sudo apt-get install libsgx-epid libsgx-quote-ex libsgx-dcap-ql
```
of course, without `libsgx-epid`

Reply:
ok, add the TODO